adoc/advanced_topics_image_management.adoc

```diff
@@ -377,7 +377,9 @@ For more information about the Kiwi build system, see the https://doc.opensuse.o
 [[at.images.kiwi.requirements]]
 === Requirements
 
-The Kiwi image building feature is available for Salt minions running {sls}{nbsp}12 or later.
+The Kiwi image building feature is available for Salt minions running {sls}{nbsp}12.
+Later, if you want to build {sle} 15 images, the Kiwi image build system must run on a {sle} 15 build host.
+Only on a {sle} 15 build host the needed version of the kiwi software will be available.
```

Comment:
Update word order:
"The required version of Kiwi will only be available on a {sle} 15 build host."

Reply:
dropped, but we must indicate that thus far it is not available.

Reply:
word order updated